Formula/px4-sim-gazebo-classic.rb

```diff
@@ -0,0 +1,39 @@
+class XRequirement < Requirement
+  fatal true
+
+  satisfy(build_env: false) { which("xquartz") }
+
+  def message
+    <<~EOS
+      XQuartz is required; install it via:
+        brew install --cask xquartz
+    EOS
+  end
+end
+
+class Px4SimGazeboClassic < Formula
+  desc "PX4 Gazebo simulation"
+  homepage "http://px4.io"
+  url "https://raw.githubusercontent.com/PX4/PX4-Autopilot/master/Tools/px4.py"
+  version "1.11.0"
```

Comment:
how is this version defined?

Reply:
getting px4.py from main branch, so no hardcoded version is needed I think.

Reply:
removed.